tests/gold_tests/pluginTest/s3_auth/gold/s3_auth_basic.gold

```diff
@@ -0,0 +1,34 @@
+*   Trying 127.0.0.1:``...
+* Connected to 127.0.0.1 (127.0.0.1) port `` (#0)
+> GET / HTTP/1.1
+> Host: www.example.com
+> User-Agent: curl/``
+> Accept: */*
+> 
+* Mark bundle as not supporting multiuse
+< HTTP/1.1 200 OK
+< Date: ``
+< Age: 0
+< Transfer-Encoding: chunked
+< Connection: keep-alive
+< Server: ATS/9.2.0
```

Comment:
We'll want to update this so it accommodates other releases, otherwise this test will fail when run on different branches.:

```
< Server: ``
```

Reply:
Nice catch!  I'll make this change.